deltaspike/modules/jsf/pom.xml

```diff
@@ -39,6 +39,62 @@
         <module>impl</module>
     </modules>
 
+    <build>
```

Comment:
if you remove this config, i would merge it
we never had this and we would need a discussion; i'm not a fan of it.

Reply:
Hi @tandraschko,
I just removed the changes to the POMs in `parent` and `modules/jsf`.

If anyone likes the idea of enforcing those consistency rules, the proposed changes are still available in https://github.com/cmunier/deltaspike/tree/DELTASPIKE-1473-enforce-consistency.

I have had a good experience with using both enforced scopes and especially "dependency analyze" in several projects, as it is a measure that is low-effort and can be used selectively in part of a multi-module project.